### ISSUE TITLE
Extended dots bug

### ISSUE THREAD

**Issue by AlexanderYeskin-azur** [mohn93/AdvancedPageControl#10]
I wasted 1.5 hours and got this result. Nice pod, well tested

https://user-images.githubusercontent.com/73464614/113872852-f1203e00-97bc-11eb-9ea9-37c94a513b87.MP4

**Comment by mohn93** [comment]
@AlexanderYeskin-azur  thanks for trying it out

Can u please send me ur setup code for this?

**Comment by AlexanderYeskin-azur** [comment]
I just used it by pod install.
Setup code the same as in example.
I tried out the example and it works well.

**Comment by AlexanderYeskin-azur** [comment]
let pageControl = AdvancedPageControlView()

pageControl.drawer = ExtendedDotDrawer(dotsColor: .gray,
                                               isBordered:true,
                                               borderColor: .white)

**Comment by mohn93** [comment]
Can u please share the code from the entire view controller setup?

**Comment by AlexanderYeskin-azur** [comment]
There is no useful information

**Comment by AlexanderYeskin-azur** [comment]
Try create your custom page control with code

**Comment by stevevovchyna** [comment]
@AlexanderYeskin-azur there's no need to be so passive-aggressive, pal. I've encountered the same issue. Just set the background color of the AdvancedPageControlView() and bug disappears.

@mohn93 thank you for the cool pod - saved me a lot of time!
I suggest setting default background color for the ExtendedDotDrawer as it behaves unexpectedly when no background color is set. Maybe the issue is that I've set it up in code 🤔
